assets/javascripts/kitten/components/steppers/vertical-stepper/components/status.js

```diff
@@ -1,57 +1,89 @@
+import classNames from 'classnames'
 
 export class Status extends Component {
-  static propTypes = {
-    valid: PropTypes.bool,
-    success: PropTypes.bool,
-    error: PropTypes.bool,
-    waiting: PropTypes.bool,
-    disabled: PropTypes.bool,
-  }
-
-  static defaultProps = {
-    valid: false,
-    success: false,
-    error: false,
-    waiting: false,
-    disabled: false,
-  }
-
   render() {
-    const { children, ...other } = this.props
+    const {
+      children,
+      valid,
+      success,
+      error,
+      waiting,
+      disabled,
+      bridge,
+      variant,
+      ...other
+    } = this.props
 
     return (
-      <StyledStatus {...other}>{this.iconByStatus() || children}</StyledStatus>
+      <StyledOrionStatus className={classNames(
+        `k-Steppers--VerticalStepper__${variant}--status`,
+      )}>
+        <StyledStatus 
+          {...other}
+          className={classNames(
+            {
+              'k-Steppers--VerticalStepper__status--valid': valid,
+              'k-Steppers--VerticalStepper__status--success': success,
+              'k-Steppers--VerticalStepper__status--error': error,
+              'k-Steppers--VerticalStepper__status--waiting': waiting,
+              'k-Steppers--VerticalStepper__status--disabled': disabled,
+              'k-Steppers--VerticalStepper__status--bridge': bridge,
+            },
+            `k-Steppers--VerticalStepper__status--${variant}`,
+          )}
+        >
+          {this.iconByStatus() || children}
+        </StyledStatus>
+      </StyledOrionStatus>
     )
   }
 
   iconByStatus = () => {
-    const { valid, success, error, waiting, disabled } = this.props
+    const { valid, success, error, waiting, disabled, variant } = this.props
+
+    if (success)
+    return <CheckedIcon width="10" title={null} />
+
+    if (error)
+      return <WarningIcon color={COLORS.error} title={null} />
 
-    if (valid) return <CheckedIcon width="10" title={null} />
-    if (success) return <CheckedIcon width="10" title={null} />
-    if (error) return <WarningIcon color={COLORS.error} title={null} />
-    if (waiting)
+    if (waiting && variant === 'andromeda')
       return <WaitingIcon height="4" color={COLORS.primary1} title={null} />
-    if (disabled)
+      
+    if (disabled && variant === 'andromeda')
       return <LockIcon width="10" color={COLORS.background1} title={null} />
 
+    if (valid && variant === 'orion') {
+      return <DotIcon width="6" color={COLORS.primary1} title={null} />
+    } else if (variant === 'andromeda') {
+      return <CheckedIcon width="10" color={COLORS.background1} title={null} />
+    }
+
     return null
   }
 }
 
 const STATUS_SIZE = 30
+const INACTIVE_STATUS_SIZE = 15
 
-export const StyledStatus = styled.span`
+const StyledOrionStatus = styled.div`
+  &.k-Steppers--VerticalStepper__orion--status {
+    border: ${pxToRem(6)} solid ${COLORS.background1};
+  }
+`
+
+const StyledStatus = styled.span`
```

Comment:
au final ça fait :

```suggestion
const StyledStatusContainer = styled.div`
  &.k-Steppers--VerticalStepper__statusContainer--orion {
    border: ${pxToRem(6)} solid ${COLORS.background1};
  }

  .k-Steppers--VerticalStepper__status {
```

sans oublier le `}` à la fin